composer.json

```diff
@@ -41,35 +41,35 @@
 	"require": {
 		"php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
 		"psr/http-message": "^1.0.1",
-		"typo3/cms-core": "^12.4.31",
-		"typo3/cms-extbase": "^12.4.31",
-		"typo3/cms-fluid": "^12.4.31",
-		"typo3/cms-frontend": "^12.4.31"
+		"typo3/cms-core": "^12.4",
+		"typo3/cms-extbase": "^12.4.31 || ^13.4",
+		"typo3/cms-fluid": "^12.4.31 || ^13.4",
+		"typo3/cms-frontend": "^12.4.31 || ^13.4"
 	},
 	"require-dev": {
 		"ergebnis/composer-normalize": "2.48.1",
 		"friendsofphp/php-cs-fixer": "3.87.1",
 		"helmich/typo3-typoscript-lint": "^3.3.0",
-		"icanhazstring/composer-unused": "0.8.11",
+		"icanhazstring/composer-unused": "0.8.11 || 0.9.3",
 		"php-parallel-lint/php-parallel-lint": "1.4.0",
 		"phpmd/phpmd": "2.15.0",
 		"phpstan/extension-installer": "1.4.3",
-		"phpstan/phpstan": "1.12.28",
-		"phpstan/phpstan-phpunit": "1.4.2",
-		"phpstan/phpstan-strict-rules": "1.6.2",
+		"phpstan/phpstan": "1.12.28 || 2.1.19",
+		"phpstan/phpstan-phpunit": "1.4.2 || 2.0.7",
+		"phpstan/phpstan-strict-rules": "1.6.2 || 2.0.6",
 		"phpunit/phpunit": "10.5.53",
-		"rector/type-perfect": "1.0.0",
-		"saschaegerer/phpstan-typo3": "1.10.2",
+		"rector/type-perfect": "1.0.0 || 2.1.0",
+		"saschaegerer/phpstan-typo3": "1.10.2 || 2.1.0",
 		"seld/jsonlint": "1.11.0",
 		"spaze/phpstan-disallowed-calls": "4.6.0",
-		"ssch/typo3-rector": "2.15.2",
-		"ssch/typo3-rector-testing-framework": "2.0.1",
-		"symfony/console": "6.4.25 || 7.3.3",
-		"symfony/translation": "6.4.24 || 7.3.3",
-		"symfony/yaml": "6.4.25 || 7.3.3",
-		"tomasvotruba/cognitive-complexity": "0.2.3",
-		"tomasvotruba/type-coverage": "1.0.0",
-		"typo3/cms-fluid-styled-content": "^12.4.31",
+		"ssch/typo3-rector": "2.15.2 || 3.1.1",
+		"ssch/typo3-rector-testing-framework": "2.0.1 || 3.0.0",
+		"symfony/console": "6.4.25 || 7.3.1",
+		"symfony/translation": "6.4.24 || 7.3.1",
+		"symfony/yaml": "6.4.25 || 7.3.1",
```

Comment:
We have update the development dependencies (including the Symfony development dependencies) in #1781. So these all should have 7.3.3 as highest version, not 7.3.1 anymore.